integration_tests/pages/bespokeConditions.ts

```diff
@@ -21,6 +21,16 @@ export default class BespokeConditionsPage extends Page {
     return this
   }
 
+  checkDeleteThisCondition = (): BespokeConditionsPage => {
+    cy.get('.delete-condition-button').should('have.text'.trim().replace(/[\r\n]+/g, ''), '\n  Delete this condition\n')
+    return this
+  }
+
+  checkDeleteTheseConditions = (): BespokeConditionsPage => {
+    cy.get('.delete-condition-button').should('have.text'.trim().replace(/[\r\n]+/g, ''), 'Delete these conditions')
```

Comment:
not sure this is right?  
`'have.text'.trim().replace(/[\r\n]+/g, '')`

you're trimming the text `have.text`

Reply:
have.text will get the text dynamically form button and then we are comparing.
I am getting some spaces and new lines so I did it.

Reply:
It is just to trim the dynamic string.

Reply:
removed it and replaced have.text to contain.text